client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -328,8 +328,10 @@ export type MonitorSettingsContact = Partial< MonitorSettingsEmail > &
 
 export type AllowedMonitorContactActions = 'add' | 'verify' | 'edit' | 'remove';
 
+export type AllowedMonitorContactTypes = 'email' | 'sms';
```

Comment:
I also added allowable contact types and set them to 'email' and 'sms' to match our endpoint. This makes sure we are using the correct strings.